SPTDataLoader/SPTDataLoaderServerTrustPolicy.m

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (c) 2015-2016 Spotify AB.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+#import "SPTDataLoaderServerTrustPolicy.h"
+#import <Security/Security.h>
+
+static BOOL SPTEvaluteTrust(SecTrustRef trust) {
+    BOOL isValid = NO;
+    SecTrustResultType result;
+    OSStatus status = SecTrustEvaluate(trust, &result);
+    if (status == errSecSuccess) {
+        isValid = (result == kSecTrustResultUnspecified || result == kSecTrustResultProceed);
+    }
+    return isValid;
+}
+
+static NSArray * SPTCertificatesForTrust(SecTrustRef trust) {
+    CFIndex count = SecTrustGetCertificateCount(trust);
+    NSMutableArray *certificates = [NSMutableArray arrayWithCapacity:(NSUInteger)count];
+    
+    for (CFIndex i = 0; i < count; i++) {
+        SecCertificateRef certificate = SecTrustGetCertificateAtIndex(trust, i);
+        [certificates addObject:(__bridge_transfer NSData *)SecCertificateCopyData(certificate)];
+    }
+    
+    return [NSArray arrayWithArray:certificates];
+}
+
+@interface SPTDataLoaderServerTrustPolicy ()
+
+@property  (strong, nonatomic) NSDictionary<NSString *, NSArray<NSData *> *> *trustedHostsAndCertificates;
```

Comment:
Might be good to switch around nonatomic/strong so it conforms to the style of the rest of the repository